manifests/sysbench/prepare.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: sysbench-prepare
+spec:
+  template:
+    spec:
+      restartPolicy: OnFailure
+      initContainers:
+      - name: set-global-variable
+        image: uhub.ucloud.cn/pingcap/sysbench:latest
+        command:
+        - /bin/sh
+        - -c
+        - |-
+          MYSQL_OPTS="-h${TIDB_HOST} -P${TIDB_PORT} -u${TIDB_USER}"
+          if [ -n "$TIDB_PASSWORD" ]; then
+            MYSQL_OPTS+=" -p${TIDB_PASSWORD}"
+          fi
+          mysql $MYSQL_OPTS -Nse 'set @@global.tidb_hashagg_final_concurrency=1;'
+          mysql $MYSQL_OPTS -Nse 'set @@global.tidb_hashagg_partial_concurrency=1;'
+          mysql $MYSQL_OPTS -Nse 'set @@global.tidb_disable_txn_auto_retry=0;'
```

Comment:
This is a side-effecting operation? After sysbench is run the configuration will be changed.
The approach I try to take is to alter the TiDB configuration to the desired production settings and then to not make any special configuration changes for the purpose of the benchmark.

Reply:
I guess it is, it is taken from this benchmark report: https://pingcap.com/docs/v3.0/benchmark/sysbench-v4/

it seems not good to alter database in these general benchmark jobs, I'll remove it

Reply:
The documentation here can point to advice for configuration. It is also possible to add an opt-in option to set this configuration.